lib/landing_session.js

```diff
@@ -213,7 +213,7 @@ class LandingSession extends Session {
     cli.log('To finish landing:');
     cli.log(`1. Run \`git push ${upstream} ${branch}\``);
     const url = `https://github.com/${owner}/${repo}/pull/${prid}`;
-    cli.log(`2. Post in ${url}: \`Landed in ${willBeLanded}\``);
+    cli.log(`2. Post \`Landed in ${willBeLanded}\` in ${url}`);
```

Comment:
```suggestion
    cli.log(`2. Post "Landed in ${willBeLanded}" in ${url}`);
```

While you're here. This isn't code.